flink-connectors/flink-connector-aws-base/src/test/java/org/apache/flink/connector/aws/testutils/LocalstackContainer.java

```diff
@@ -61,6 +61,11 @@ public String getEndpoint() {
 
         @Override
         protected void waitUntilReady() {
+            try {
+                Thread.sleep(10000);
+            } catch (InterruptedException e) {
+                e.printStackTrace();
```

Comment:
Understand this is the same as the KDS one, but if we are interrupted should probably exit. Instead of just printing can we rethrow? Worth updating the KDS one too

Reply:
Sounds good